packages/@react-spectrum/tabs/src/Tabs.tsx

```diff
@@ -325,7 +325,7 @@ export function TabPanels<T>(props: SpectrumTabPanelsProps<T>) {
 
   return (
     <FocusRing focusRingClass={classNames(styles, 'focus-ring')}>
-      <div {...styleProps} {...tabPanelProps} ref={ref} className={classNames(styles, 'spectrum-TabsPanel-tabpanel')}>
+      <div {...styleProps} {...tabPanelProps} ref={ref} key={tabPanelProps.id} className={classNames(styles, 'spectrum-TabsPanel-tabpanel')}>
```

Comment:
If I read the code correctly the user cannot define this key. TabPanel doesn't accept keys in the prop and each panel needs a key, not the panel key. Items can take a key and it is added to the tabPanelProps.id value. And I don't think I have access to the children of the items to add it on each field.